sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyServiceClient.java

```diff
@@ -0,0 +1,159 @@
+package com.azure.security.keyvault.keys.cryptography;
```

Comment:
Licensing header

Reply:
Fixed,
But checkstyle fixes coming in the batch at the end today, assume they don't exist.